code/modules/admin/verbs/adminhelp.dm

```diff
@@ -345,6 +346,8 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 	if(initiator && initiator.current_ticket == src)
 		initiator.current_ticket = null
 
+	SEND_SIGNAL(src, COMSIG_ADMIN_HELP_INACTIVE)
```

Comment:
Can you place the signal at the start of the proc, after the checks that'd make it return early?
Signals are better sent as early as possible.
And if not, can you put in the signal name that it's reporting a post-fact? Something like "removed", "made inactive" or similar.

Reply:
`Signals are better sent as early as possible.` can you post some actual reasoning for this because I never heard / have forgotten why that's actually true. The only actual thing you should care about with signals is that it signifies an event.